README.md

```diff
@@ -17,56 +17,170 @@
+beginning of the semester. If you're unsure if it's set up and
+working correctly, try:
+
+```bash
+mongo
+```
+
+If your MongoDB server isn't installed you'll likely get an error
+message like:
+
+```text
+Error: couldn't connect to server 127.0.0.1:27017, connection attempt failed: SocketException: Error connecting to 127.0.0.1:27017 :: caused by :: Connection refused :
+```
+
+(Use CTRL-D to exit out of the `mongo` tool.)
+
+### Open the project in VS Code
+
+Launch Visual Studio Code, and then choose `File -> Open Folder…`. Navigate to your clone
+of the repo and choose `Open`.
+
+You may see a dialog that looks like this if you don't already have the recommended extensions:
+
+![Dialog suggesting installation of recommended extensions](https://user-images.githubusercontent.com/1300395/72710961-bf767500-3b2d-11ea-8ea4-fbbd39c78da5.png)
+
+Don't worry if you don't get the dialog, it is probably because you already have them all installed.
+
+Like in previous labs, click "Install All" to automatically install them.
+
+### Installing the client dependencies
+
+Before you start working you will need to install the dependencies for the client.
+
+1. Move into the `client` directory (`cd client`)
+2. Run `npm install`
+
+### Enable ESLint in VS Code
+
+ESLint is a tool for checking the quality and style of your TypeScript
+and JavaScript code, and can provide valuable "live" feedback on
+your coding in VS Code.
+
+Since this is the first time we will be using ESLint there is an additional step to make sure the VS Code extension is working in the project. When you first open a TypeScript file you will see at the bottom right that ESLint is disabled.
+
+![image](https://user-images.githubusercontent.com/1300395/107999308-bc59ec80-6fac-11eb-9784-75a471a50aa4.png)
+
+Click the red "ESLINT" to open this dialog:
+
+![image](https://user-images.githubusercontent.com/1300395/107996971-528b1400-6fa7-11eb-89bc-afc71747f820.png)
+
+Click "Allow Everywhere" to enable ESLint.
+
+You can also open this dialog with the following steps:
+
+1. Hit `CTRL + SHIFT + P` (`⌘ + ⇧ + P` on Macs) to open the Command Palette. You can also find this by going to the "View" menu and clicking "Command Palette..."
+2. Start typing and select "ESlint: Manage Library Execution". That should open a dialog seen above.
+
+### Seeding the Database
+
+To give yourself some data to work with instead of starting with an empty database in our development environment, you need to 'seed' the database with some starter data. Seed data and the seed script are stored in the top level directory `database`. To seed the database, move into that directory and run `./mongoseed.sh` (or `.\mongoseed.bat` on Windows). This will take each of the JSON files in `database/seed/` and insert their elements into the `dev` database. (If you want to
+seed a different database, you can provide the name of the desired
+database as an argument to the `mongoseed` script.).
```

Comment:
```suggestion
To give yourself some data to work with instead of starting with an empty database in our development environment, you need to 'seed' the database with some starter data. Seed data and the seed script are stored in the top level directory `database`. To seed the database, move into that directory and run `./mongoseed.sh` (or `.\mongoseed.bat` on Windows). This will take each of the JSON files in `database/seed/` and insert their elements into the `dev` database.
```